packages/@glimmer/util/lib/simple-cast.ts

```diff
@@ -244,22 +101,26 @@ function isElement(node: Node | SimpleNode | SimpleElement): node is Element | S
   return node.nodeType === NodeType.ELEMENT_NODE;
 }
 
-function isBrowserNode(node: Node | SimpleNode): node is Node {
-  return typeof document !== undefined && node.ownerDocument === document;
-}
+export function checkNode<S extends SugaryNodeCheck>(
```

Comment:
Just to be clear, this is a `DEBUG` only utility (or actually, it may be a `LOCAL_DEBUG`?) right?

Reply:
Yes, it is LOCAL_DEBUG